metagraph/src/graph/graph_extensions/node_rc.cpp

```diff
@@ -32,6 +32,14 @@ NodeRC::NodeRC(const DeBruijnGraph &graph, bool construct_index) : graph_(&graph
         return;
 
     const BOSS &boss = dbg_succ->get_boss();
+    const auto *mask = dbg_succ->get_mask();
+    std::unique_ptr<bit_vector_stat> computed_mask;
+    if (!mask) {
+        auto sdsl_mask = boss.mark_all_dummy_edges(get_num_threads());
+        sdsl_mask.flip();
+        computed_mask = std::make_unique<bit_vector_stat>(std::move(sdsl_mask));
+        mask = computed_mask.get();
```

Comment:
No need to set it, just use the sdsl bitmap.
Actually if you do set it, it changes the node indexes

Reply:
this isn't setting it, actually, it's only using what's computed by `mark_all_dummy_edges`.

If it were setting the mask, this would not compile